### DISCUSSION TITLE
What does the "S" in the top left corner mean?

### DISCUSSION THREAD

**Discussion by DurvalMenezes** [ludwich66/Quansheng_UV-K5_Wiki#12] [Q&A] [answered]
As per the pic: 
![pic](https://github.com/ludwich66/Quansheng_UV-K5_Wiki/assets/5405638/0cfbc472-e9ad-4247-981d-908b9e827316)

The manual doesn't mention it.

I noticed it turns on and off by itself while listening, but when on, always turns off when the PTT is pressed.

Wild guess: maybe it indicates "S"quelching is occurring?

TIA!

**Comment by troilus** [comment]
Into power save mode.

**Comment by DurvalMenezes** [comment]
> Into power save mode.

Hummmrmrmr... are you sure about that? because if it was, I think it would turn on and off periodically with a frequency determined by the "SAVE" menu item, right? Like 1:1 would be 50% off and 50% on, etc. 

But instead, what I see is it turning on and off and very random intervals -- and when I turned squelch ("SQL" menu item) down from 4 to 1, it seemed to occur less frequently.

**Reply by troilus** [reply, marked as answer]
I asked official after-sales again, and they still answered me as "power saving symbol".

**Reply by DurvalMenezes** [reply]
Thanks for re/checking with them, @troilus. Marking yours as the correct answer.

**Reply by Tunas1337** [reply]
To clarify: the "S" just means that the radio is _now executing_ the power-saving routine, in general - it won't show you when the BK4819 radio chip is actually on or off, it just means that when the icon is fired up, it is doing the cycling in the background. I have verified this by outputting the chip's power state once every 100ms.

**Reply by ludwich66** [reply]
And "NS" for "NOAA SCAN" uses the same Position?
